joint_names:
  - shoulder_yaw_joint_left
  - shoulder_pitch_joint_left
  - shoulder_roll_joint_left
  - elbow_pitch_joint_left
  - elbow_roll_joint_left
  - wrist_pitch_joint_left
  - wrist_yaw_joint_left
constraints:
   goal_time: 1.0
   shoulder_yaw_joint_left:
        goal: 0.06
        trajectory: 1.0
   shoulder_pitch_joint_left:
        goal: 0.06
        trajectory: 1.0
   shoulder_roll_joint_left:
        goal: 0.06
        trajectory: 1.0
   elbow_pitch_joint_left:
        goal: 0.06
        trajectory: 1.0
   elbow_roll_joint_left:
        goal: 0.06
        trajectory: 1.0
   wrist_pitch_joint_left:
        goal: 0.06
        trajectory: 1.0
   wrist_yaw_joint_left:
        goal: 0.06
        trajectory: 1.0
